packages/modules/src/modules/banners/investigationsMoment/InvestigationsMomentBanner.tsx

```diff
@@ -149,7 +149,6 @@ function InvestigationsMomentBanner({
                     mobileHeading={content.mobileContent?.heading ?? null}
                 />
             </div>
-
```

Comment:
Could keep the spaces, does make these components nicer to read

Reply:
Ok, put line spaces back